catroid/src/main/java/org/catrobat/catroid/ui/adapter/BrickAdapter.java

```diff
@@ -93,20 +91,17 @@
 	private int toEndDrag;
 	private boolean retryScriptDragging;
 	private boolean showDetails = false;
+	public boolean isDragging = false;
```

Comment:
this is only a temporary fix and should be changed asap. I'll have another look at it today so it might no longer be necessary anyway.